source/ota.c

```diff
@@ -3186,6 +3186,12 @@ OtaState_t OTA_Shutdown( uint32_t ticksToWait,
     {
         otaAgent.unsubscribeOnShutdown = unsubscribeFlag;
 
+        /* Stop the request timer. */
```

Comment:
NIT: consider moving to the end of this block after actual shutdown.

Reply:
This needs to occur before shutdown is queued so that it will always happen before the struct is cleared,